server/lib/notifications.js

```diff
@@ -201,14 +201,48 @@ async function notifyByEmail(activity) {
 
     case activityType.COLLECTIVE_UPDATE_PUBLISHED:
       twitter.tweetActivity(activity);
-      activity.data.update = await models.Update.findByPk(activity.data.update.id, {
-        include: [{ model: models.Collective, as: 'fromCollective' }],
-      });
-      activity.data.update = activity.data.update.info;
-      notifyMembersOfCollective(activity.data.update.CollectiveId, activity, {
-        from: `${activity.data.collective.name}
-        <no-reply@${activity.data.collective.slug}.opencollective.com>`,
-      });
+
+      activity.data.collective = await models.Collective.findByPk(activity.data.collective.id);
+      activity.data.hostedCollectives = activity.data.collective.getHostedCollectives();
+
+      if (activity.data.collective.info.isHostAccount) {
+        switch (activity.data.update.updateNotificationAudience) {
+          case 'Hosted Collective Admins':
+            activity.data.hostedCollectives.map(collective => {
+              notifyAdminsOfCollective(collective.id, activity, {
+                from: `${activity.data.collective.name}
+                <no-reply@${activity.data.collective.slug}.opencllective.com>`,
```

Comment:
This `from` param is alway the same, maybe we can extract the full object to an `emailParams` variable?